docs/start_3d.md

```diff
@@ -91,6 +91,7 @@ What to configure:
 - Length of marker separation (for aruco) or square side (for charuco or checkerboard) (triangulation is set to this unit)
 - Length of marker side in appropriate unit, in same unit as above
 - aruco marker dictionary (number of bits and number of markers in dictionary)
+- Boolean value indicating whether or not you want to manually verify the detection of the calibration pattern in each frame (Allows you to throw out bad detections)
```

Comment:
It should be clear that specifying this is optional (with default false) and specified in its own section (e.g. "Verify detections manually") rather than in the "Calibration marker configuration" section.